index.js

```diff
@@ -116,6 +116,10 @@ function _supportsColor(haveStream, {streamIsTTY, sniffFlags = true} = {}) {
 		return /^(9\.(0*[1-9]\d*)\.|\d{2,}\.)/.test(env.TEAMCITY_VERSION) ? 1 : 0;
 	}
 
+	if ('TF_BUILD' in env && 'AGENT_NAME' in env) {
```

Comment:
This needs a code comment what product this check if for.

Reply:
```suggestion
	// Check for Azure DevOps pipelines
	if ('TF_BUILD' in env && 'AGENT_NAME' in env) {
```